webapi/src/test/scala/org/knora/webapi/e2e/v2/OntologyV2R2RSpec.scala

```diff
@@ -1824,6 +1823,207 @@ class OntologyV2R2RSpec extends R2RSpec {
+           |    "knora-api" : "http://api.knora.org/ontology/knora-api/v2#",
+           |    "owl" : "http://www.w3.org/2002/07/owl#",
+           |    "xsd" : "http://www.w3.org/2001/XMLSchema#",
+           |    "rdfs" : "http://www.w3.org/2000/01/rdf-schema#"
+           |  }
+           |}""".stripMargin
+
+      Put("/v2/ontologies/properties", HttpEntity(RdfMediaTypes.`application/ld+json`, params)) ~> addCredentials(
+        BasicHttpCredentials(anythingUsername, password)
+      ) ~> ontologiesPath ~> check {
+        val responseStr = responseAs[String]
+
+        assert(status == StatusCodes.OK, response.toString)
+        val responseJsonDoc    = JsonLDUtil.parseJsonLD(responseStr)
+        val updatedOntologyIri = responseJsonDoc.body.value("@id").asInstanceOf[JsonLDString].value
+        assert(updatedOntologyIri == ontoIri)
+        val graph               = responseJsonDoc.body.requireArray("@graph").value
+        val property            = graph.head.asInstanceOf[JsonLDObject]
+        val returnedPropertyIri = property.requireString("@id")
+        returnedPropertyIri should equal(propertyIri)
+        val returnedComment = property.requireObject(OntologyConstants.Rdfs.Comment).requireString("@value")
+        returnedComment should equal(newComment)
+        val lastModDate = responseJsonDoc.body.requireDatatypeValueInObject(
+          key = OntologyConstants.KnoraApiV2Complex.LastModificationDate,
+          expectedDatatype = OntologyConstants.Xsd.DateTimeStamp.toSmartIri,
+          validationFun = stringFormatter.xsdDateTimeStampToInstant
+        )
+
+        assert(lastModDate.isAfter(fooLastModDate))
+        anythingLastModDate = lastModDate
+
+        // load back the ontology to verify that the updated property still is editable
+        val encodedIri = URLEncoder.encode(s"${SharedOntologyTestDataADM.ANYTHING_ONTOLOGY_IRI_LocalHost}", "UTF-8")
+        Get(
+          s"/v2/ontologies/allentities/${encodedIri}"
+        ) ~> ontologiesPath ~> check {
+          val responseStr: String = responseAs[String]
+          assert(status == StatusCodes.OK, response.toString)
+          val responseJsonDoc = JsonLDUtil.parseJsonLD(responseStr)
+
+          val graph           = responseJsonDoc.body.requireArray("@graph").value.map(_.asInstanceOf[JsonLDObject])
+          val nothingValue    = graph.filter(_.requireString("@id") == propertyValueIri).head
+          val isEditableMaybe = nothingValue.maybeBoolean(OntologyConstants.KnoraApiV2Complex.IsEditable)
+          isEditableMaybe should equal(Some(true))
+        }
+      }
+    }
+
+    "delete the property comment and check if the property is still correctly marked as `isEditable: true`" in {
+      // update label and comment of a property
+      val ontoIri            = SharedOntologyTestDataADM.ANYTHING_ONTOLOGY_IRI_LocalHost
+      val propertyIri        = ontoIri + "#hasOtherNothing"
+      val propertyValueIri   = ontoIri + "#hasOtherNothingValue"
```

Comment:
Cannot these 3 be defined once in the entire file and reused instead of dining them in every test?